pkg/generator/changelog_types.go

```diff
@@ -30,51 +31,63 @@ func (ct *ChangelogTypes) AppendContent(cType, content string) {
 
 var defaultTypes = ChangelogTypes{
 	{
-		Type: "%%bc%%",
-		Text: "Breaking Changes",
+		Type:  "%%bc%%",
+		Text:  "Breaking Changes",
+		Emoji: "📣 ",
```

Comment:
Can we remove the space character here for all fields and add it when needed with `prettifyPrefix = ct.Emoji + " "`

Reply:
will do